panther_hardware_interfaces/src/panther_system/robot_driver/lynx_robot_driver.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "panther_hardware_interfaces/panther_system/robot_driver/lynx_robot_driver.hpp"
+
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "panther_hardware_interfaces/panther_system/robot_driver/roboteq_driver.hpp"
+#include "panther_hardware_interfaces/panther_system/robot_driver/roboteq_robot_driver.hpp"
+
+namespace panther_hardware_interfaces
+{
+
+void LynxRobotDriver::SendSpeedCommands(const std::vector<float> & speeds)
+{
+  if (speeds.size() != 2) {
+    throw std::runtime_error(
+      "Invalid speeds vector size. Expected 2, got " + std::to_string(speeds.size()));
+  }
+
+  const auto speed_left = this->GetCmdVelConverter().Convert(speeds.at(0));
```

Comment:
Can we use here `MotorChannels::LEFT` and `MotorChannels::Right` instead of 0 and 1?

Reply:
Channels are left: `2` and right: `1` so it won't work. The order of velocities is explained in method brief.